src/Entity/ReferentManagedUsersMessage.php

```diff
@@ -0,0 +1,196 @@
+<?php
+
+namespace AppBundle\Entity;
+
+use Algolia\AlgoliaSearchBundle\Mapping\Annotation as Algolia;
+use AppBundle\Referent\ReferentMessage;
+use Doctrine\ORM\Mapping as ORM;
+use Ramsey\Uuid\UuidInterface;
+
+/**
+ * @ORM\Table(name="referent_managed_users_message")
+ * @ORM\Entity(repositoryClass="AppBundle\Repository\ReferentManagedUsersMessageRepository")
+ *
+ * @Algolia\Index(autoIndex=false)
+ */
+class ReferentManagedUsersMessage
+{
+    use EntityIdentityTrait;
+    use EntityTimestampableTrait;
+
+    /**
+     * @var Adherent
```

Comment:
The mapping implies that it might be null in the future if the adherent is deleted.

Reply:
not anymore: `onDelete="CASCADE"`